add maintainers-minutes-2025-05-30 (#3377)
nf-core/website: sites/main-site/src/content/blog/2025/maintainers-minutes-2025-05-30.mdx

---
title: "Maintainers Minutes: May 2025"
subtitle: "Keeping you informed of the latest maintainers discussions"
pubDate: 2025-06-03T10:00:00+01:00
headerImage: "/assets/images/blog/maintainers-minutes-2024-07-26/maintainers-wide.png"
headerImageAlt: "Cartoon yellow rubber duck with nf-core logo badge on its body with the nf-core logo."
embedHeaderImage: true
authors:
    - "maxulysse"
label:
    - "maintainers"
---

import Profile from "@components/GitHubProfilePictureExtended.svelte";
import sad_maintainers_duck from "@assets/images/blog/maintainers-minutes-2024-06-07/sad-maintainers-duck.png";
import notifications from "@assets/images/blog/maintainers-minutes-2025-04-24/too-many-notifications.jpg";
import botoverlords from "@assets/images/blog/maintainers-minutes-2025-04-24/bot-overlords.jpg";
import saveapolarbear from "@assets/images/blog/maintainers-minutes-2025-04-24/save-a-polar-bear.jpg";
import { Image } from "astro:assets";

> The 'Maintainers Minutes' aims to give further insight into the workings of the [nf-core maintainers team](/governance#maintainers)
> by providing brief summaries of the monthly team meetings.

Reviewer: ```suggestion
The 'Maintainers Minutes' aims to give further insight into the workings of the [nf-core maintainers team](/governance#maintainers) by providing brief summaries of the monthly team meetings.
```

## Overview

What was initially planned as a meeting focused on test data was unfortunately delayed due to key members being absent.

> Maxime has made a full takeover and redirected the meeting into a presentation of the new RFC initative from the core team.

Reviewer: ```suggestion

Maxime performed a full takeover and redirected the meeting into a presentation of the recently introduced RFC initiative from the core team.
```

But before all that, Louis gave us a quick run down on the pytest to nf-test tests migration effort that he has been recently spear-heading with the help of many community members.
A tools release is on its way, coming with lots of goodness.

## Pytest to nf-test tests migration for nf-core/modules

> On the day this meeting happened, only one module and one subworkflow were left to migrate.

Reviewer: ```suggestion
As of the day the meeting was held, only one module and one subworkflow were left to migrate 😱.
```

> So we are hoping to finally close this issue that was started two years ago.

Reviewer: ```suggestion
So we are hoping to finally close this issue that was started two years ago.

A HUGE thank you :people_hugging: for everyone who contributed to this huge effort of adding nf-test to [>1500 modules](https://nf-co.re/modules/)!
```

## RFCs aka Request For Comments

> This new initative coming from the core team is meant to help formalize and present ideas to improve the nf-core.

Reviewer: ```suggestion
This new initiative coming from the core team is meant to help formalize and present substantial ideas to improve the nf-core.

```

A discussion can be started on Slack, which can lead to an issue on nf-core/proposals, which in the end can lead into a Pull Request.
For more information, please read [the documentation](https://nf-co.re/docs/contributing/project_proposals).
The important part is that we maintainers will discuss these proposals and coordinate with the core team to help port all of these to fruition.

## nf-core tools

> It is happening, and we're quite happy about it, as a lot of goodness is coming with it.

Reviewer: ```suggestion
The next update is HAPPENING and we're quite happy about it, as a lot of goodness is coming with it.
```
